internal/pkg/service/common/etcdop/op/tracker_test.go

```diff
@@ -64,6 +64,17 @@ func TestTracker(t *testing.T) {
 		Commit()
 	assert.NoError(t, err)
 
+	// Get records
+	operations := tracker.Operations()
+	for i := range operations {
+		op := &operations[i]
+		// Clear complex KVs slice, assert count
+		if op.Type == GetOp {
+			assert.Len(t, op.KVs, int(op.Count))
```

Comment:
`KVs` are comples values, so we are not asserting them, we are asserting only the items count.